jbt-it/neomdb
# 180 client create project

Added a new page to tender or save (Altes Projekt eintragen) a new project. Added a stepper to display the different steps.
The saved project will be logged to the console for now.
Also added mock data for the companies and contact persons.

How to test:
- go to /projekte and click on "PROJEKT AUSSCHREIBEN"
- Enter differnt date and check for error handling, e.g. check if the max can be set to lower than the min
- Check if leaving the application end empty will lead you to the saving overview and entering a valid application end will lead you to tendering the project
- "Create" a new customer and see if the data is saved to the console (The customer will later be saved to the db and will then be availabel in the autocomplete)
